e2e-agent/create.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import os
+from typing import Optional
+
+import requests
+from dotenv import load_dotenv
+
+SYSTEM_PROMPT_TEMPLATE = """\
+You are an interviewer for candidates applyine to a role.
+You ask three relevant questions and then end the conversation.
+
+This is the role to interview for: {role_name}
+Role description: {role_description}
+The name of the candidate is: {candidate_name}
+"""
+
+EGE_STOCK_AVATAR_ID = "b9be11b8-89fb-4227-8f86-4a881393cbdb"
+API_URL = "https://api.bey.dev/v1"
+
+
+def main(
+    api_key: str,
+    role_name: str,
+    role_description: str,
+    candidate_name: str,
+    avatar_id: Optional[str],
+) -> None:
+    response = requests.post(
+        f"{API_URL}/agent",
+        headers={"x-api-key": api_key},
+        json={
+            "name": f"{role_name} Interviewer for {candidate_name}",
+            "system_prompt": SYSTEM_PROMPT_TEMPLATE.format(
+                role_name=role_name,
+                role_description=role_description,
+                candidate_name=candidate_name,
+            ),
+            "greeting": (
+                f"Hello {candidate_name}, how are you doing? "
+                "Are you ready for the interview?"
+            ),
+            "avatar_id": avatar_id if avatar_id is not None else EGE_STOCK_AVATAR_ID,
+            # ---
+            # Uncomment the following lines to customize the avatar further
+            # "language": "es",
+            # "max_session_length_minutes": 10,
+            # "capabilities": ["webcam_vision"],
```

Comment:
Maybe some comments nice to explain what these do where not obvious?

```suggestion
            # Uncomment the following lines to customize the avatar further
            # "language": "es",  # switch conversation language to Spanish
            # "max_session_length_minutes": 10,
            # "capabilities": ["webcam_vision"],  # enable agent to see the candidate's camera
```